codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -168,9 +168,7 @@ private int unconsumedBytes(Http2Stream stream) {
 
     void onGoAwayRead0(ChannelHandlerContext ctx, int lastStreamId, long errorCode, ByteBuf debugData)
             throws Http2Exception {
-        // Don't allow any more connections to be created.
         connection.goAwayReceived(lastStreamId, errorCode, debugData);
```

Comment:
@buchgr - Make sure you update the comments on https://github.com/netty/netty/blob/master/codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Connection.java#L100 related to this.